LCU.NET.Extras/Game/Actuator.Game.cs

```diff
@@ -234,7 +233,7 @@ public async Task UploadRunePage(Position pos, int championId)
 
             LogTo.Debug("for champion {0}", champion);
 
-            Main.ShowNotification(Text.LockedInMessage, champion + ", " + pos.ToString().ToLower(), NotificationType.Success);
+            Main.ShowNotification("Locked in", champion + ", " + pos.ToString().ToLower(), NotificationType.Success);
```

Comment:
I don't like those hardcoded strings but... i don't know how to solve this without a dependency to our system

Reply:
Yep, I was thinking of just removing the notifications, or adding i18n here. Talking with @axelgorris, he managed it in his story, and luna agrees that it's left as only english.